fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -977,6 +962,19 @@ abstract class BaseBrowserFragment :
         initializeEngineView(toolbarHeight)
     }
 
+    protected fun showUndoSnackbar(message: String) {
+        viewLifecycleOwner.lifecycleScope.allowUndo(
+            binding.dynamicSnackbarContainer,
+            message,
+            requireContext().getString(R.string.snackbar_deleted_undo),
+            {
+                requireComponents.useCases.tabsUseCases.undo.invoke()
+            },
+            paddedForBottomToolbar = true,
+            operation = { },
+        )
+    }
```

Comment:
I'd be curious having an optional follow-up (for anyone) to investigate getting `viewLifecycleOwner.lifecycleScope.allowUndo` to work in Compose previews or creating a Compose analog for us to use with future from-scratch features.

Reply:
This would be nice and useful especially as we migrate the parents (Home, Fragment) to compose, maybe already in Tabs Tray? showing/hiding the snackbar would be a result of updating the state